automation/addLabelsOnPullRequest.automation.js

```diff
@@ -170,36 +103,36 @@ exports.changesRequestLabel = async context => {
 	const reviews = await context.octokit.pulls.listReviews(params);
 
 	const changesRequestedReviews = reviews.data.filter(
-		review => review.state === 'CHANGES_REQUESTED'
+		review => review.state === "CHANGES_REQUESTED"
 	);
 
 	const foundChangedRequestedLabel = issueLabels.data.filter(
-		label => label.name === ':warning: Changes requested'
+		label => label.name === ":warning: Changes requested"
 	);
 
 	const foundReadyForReviewLabel = issueLabels.data.filter(
-		label => label.name === ':mag: Ready for Review'
+		label => label.name === ":mag: Ready for Review"
 	);
 
 	const foundApprovedLabel = issueLabels.data.filter(
-		label => label.name === ':white_check_mark: Approved'
+		label => label.name === ":white_check_mark: Approved"
 	);
 
 	if (
 		changesRequestedReviews.length > 0 &&
 		foundChangedRequestedLabel.length === 0
 	) {
-		addLabel([':warning: Changes requested'], context, 'AA2626');
+		addLabel([":warning: Changes requested"], context, "AA2626");
```

Comment:
Codacy has a fix for the issue: [Replace `↹↹` with `····`](https://app.codacy.com/gh/aayushchugh/bot/pullRequest?prid=10060852)
```suggestion
    addLabel([":warning: Changes requested"], context, "AA2626");
```